### System Parameters ###
sim: True
use_numerical_ik: False # if False, will use analyticial IK solver
show_cam_photo: False # toggle to show figure for every archival photo acquired
show_pie_chart: False # toggle to show colors identified by k-means clustering for color masks
scale_ft_mode: False # if False, will use contact mode
scale_tare: 0.1 # mass [kg] of scale cover link in simulation


working_pose: # translation between stations
  z: 0.18 
  roll: 0
  pitch: 1.55


### Pickup Parameters ###
gripper_len: 0.0775
orient_z: 0.03 # height above sherd at which gripper orients itself before descending to the table to grasp
clearance: .0025  # gap between end of fingers and table/mat [m]
sherd_allowance: 0.02 # in case sherd curves past the tips of the the gripper fingers
overlapping_sherds: True
cluster_threshold: 0.01 # if sherds overlap: max. z-distance between pixels to be considered members of the same sherd
survey_z: 0.28 # height from which to search for sherds
table_z: 0.0


### Calibration Locations ###
mat_color_location:
  x: 0.3
  y: 0.15

scale_color_location:
  x: 0.215
  y: 0


### Station Locations and Z-Heights ###
pickup_area:
  step_size: 0.05
  offset_x: -0.03
  offset_y: 0.20
  length_x: 0.35
  length_y: 0.15

scale_location:
  x: 0.3
  y: 0.0
  z: 0.015

scale_survey_location:
  x: 0.23
  y: 0.0
  
cam_location:
  x: 0.32
  y: -0.3
  z: 0.0

cam_survey_location:
  x: 0.265
  y: -0.203

standby_location:
  x: 0.26
  y: 0.0

discard_area:
  z: 0.025
  offset_x: -0.05 #-0.03
  offset_y: -0.375 #-0.35
  length_x: 0.14
  length_y: 0.13

